src/Actions/Actions.csproj

```diff
@@ -18,7 +18,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Roslyn.Diagnostics.Analyzers" Version="3.0.0">
+    <PackageReference Include="Roslyn.Diagnostics.Analyzers" Version="3.3.0">
```

Comment:
(aside): AIWin tries to centralize the analyzers in the build\NetFrameworkRelease.targets file. Is that a pattern to adopt here?

Reply:
We intentionally don't do that so that dependabot can update the dependencies

Reply:
I'm not sure this is really a problem. Dependabot successfully updated the targets file in https://github.com/microsoft/accessibility-insights-windows/pull/894